Fulfillment/Integration/01_ManageConnections.md

```diff
@@ -0,0 +1,143 @@
+
+    ![Credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/Credentials.png "[Credentials]")
+
+    > [Info] Depending on the selected driver, the fields in the *Credentials* section differ.
+
+
+4. Enter the required data in the fields in the *Credentials* section.
+
+5. Click the [SAVE] button.   
+    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+    ![Checking credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Checking credentials]")
+
+    The *Create connection* view is automatically closed when the connection has been established. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
+
+    ![Edit connection settings](../../Assets/Screenshots/Fulfillment/Settings/Connections/EditConnectionSettings.png "[Edit connection settings]")
+
+    > [Info] By default, all connections that have been established are first inactive and therefore must be enabled after the connection configuration, see [Enable a connection](#enable-a-connection).
+
+
+## Edit a connection
+
+Edit a connection to adjust any changed credentials, the connection name or further settings. The driver selection is read-only and cannot be edited.
+
+#### Prerequisites
+
+At least one connection has been created, see [Create a connection](#create-a-connection).
+
+#### Procedure
+
+*Fulfillment > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connections]")
+
+1. Click the connection you want to edit in the list of connections.   
+    The *Edit connection* view is displayed.
+
+    > [Info] Depending on the driver of the connection, the *Credentials* and the *Settings* tab are displayed in the *Edit connection* view.
+
+    ![Edit connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/EditConnectionCredentials.png "[Edit connection]")
+
+2. Edit the desired data of the connection in the corresponding fields in the *Credentials* tab.
+
+3. If necessary, click the *Settings* tab and edit the desired settings of the connection in the corresponding fields.
+
+4. Click the [SAVE] button.   
+    The *Checking credentials...* notice is displayed.
+
+    ![Credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Credentials]")
+
+    The *Edit connection* view is automatically closed and the *Connections* view is displayed again.
+
+
+## Enable a connection
+
+A connection can only be used if it is active. Otherwise, no data is being synchronized via the connection. By default, all connections that have been established are first inactive and therefore must be enabled. If a connection has been disabled temporarily, it can be enabled it again. It is possible to enable several inactive connections at a time.
```

Comment:
```suggestion
A connection can only be used if it is active. Otherwise, no data is being synchronized via the connection. By default, all connections that have been established are inactive and therefore must be enabled. If a connection has been disabled, it can be enabled again. It is possible to enable several connections at a time.
```